core/index.rst

```diff
@@ -1,6 +1,256 @@
 Tenstorrent
 ==============================
 
+.. raw:: html
```

Comment:
Could we use `:include:` tag here and have the HTML inside of `home.html`?

Reply:
Yes- much better idea- I'll do this

Reply:
Couldn't figure this out with `:include:`, but was able to move the HTML into `/core/_templates/home.html` and then include it here with:

```rst
.. raw:: html
   :file: _templates/home.html
```